tests/iDibo_tests.py

```diff
@@ -1,11 +1,24 @@
 from nose.tools import *
-import iDibo
 
-def setup():
-    print "SETUP!"
+import unittest
+from gluon.globals import Request
 
-def teardown():
-    print "TEAR DOWN!"
+db = test_db
 
-def test_basic():
-    print "I RAN!"
\ No newline at end of file
+execfile("applications/iDibo/controllers/default.py", globals())
+db(db.auth_user.id > 0).delete()
+db.commit()
+
+
+class TestRegister(unittest.TestCase):
+    def setup(self):
+        request = Request()
+
+    def test_register_title(self):
+        self.controller.register()
+        my_title = self.controller.register.response.title
```

Comment:
For debugging:
At this level, since there is no controller, you don't have access to any response object, so there is no way to verify the title. You can only verify what you return from the register function, which is dict(form=auth.register()). 
Please read about dictionaries at https://docs.python.org/2/tutorial/datastructures.html
Assuming you don't know what happened, you should check what the response looks like:
    resp = register()
        for key, value in resp.iteritems():
            print "[", key, "]: "
            print "(", value, ")\n"
1. What do you see?
2. Can you access the response title here?